prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -381,6 +381,51 @@ periodics:
           requests:
             memory: 200Mi
             cpu: 80m
+- name: kyma-gke-end-to-end-test-backup-restore
+  cron: "0 9 * * 1-5" # "At 09:00 on every day-of-week from Monday through Friday."
+  labels:
+    preset-build-master: "true"
+    preset-stability-checker-slack-notifications: "true"
```

Comment:
In kyma-gke-integration.sh we don't configure stability checker, so probably you don't need it

Reply:
removed